docker/konduit-python/run.sh

```diff
@@ -0,0 +1,9 @@
+echo "Installing required packages..."
+pip install --no-cache-dir -r requirements.txt
+echo "Python packages installed successfully."
+
+echo "-------------------------------------------------------------------"
+
+echo "Starting konduit server..."
```

Comment:
I'd allow a potential over ride in the docker container. If you want to pre package a python script execution experience, have a default script in there and allow a volume mounting an over rideable script in to the container. Maybe add a docker compose example in there along side that.

Reply:
@agibsonccc yes at the end the user just need to override the script with a volume using `-v`